internal/lib/control_config.go

```diff
@@ -252,3 +258,30 @@ func (c *akoControlConfig) PodEventf(eventType, reason, message string, formatAr
 		}
 	}
 }
+
+func GetLicenseTypeFromURI(client *clients.AviClient) (models.SystemConfiguration, error) {
```

Comment:
please take care of the name of the function as the function's name and what the function actually returns are not the same.